lib/components/Overlays/Dialog/index.tsx

```diff
@@ -59,7 +59,7 @@ const OneDialog = forwardRef<HTMLDivElement, DialogProps>(
             <DialogHeader>
               {header.icon && (
                 <DialogIcon>
-                  <header.icon size={24} />
+                  <header.icon size={32} strokeWidth={1.5} />
```

Comment:
Should we use tokens for these numbers?

Reply:
Umm, I guess we won't be using Lucide icons in the near future, but rather our own icons. Since we're yet to incorporate them into One, I've put together an example to give a taste of how an icon should shape up.